### PR TITLE
Access versioned metadata all

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# ACCESS VERSIONED METADATA ALL

metadata-service has thus far only accesed version 1.0.0 of metadata_all. This was hardcoded as an MVP. This pull requests instead reads from the metadata all requested in the "version" field of the query.

## Why?
This was written to keep up with the versioning logic written in other microdata components. It felt like the right time to upgrade this facet of the metadata-service MVP.

## How?
* Data-service already handles versioned requests for data, so this only concerns the metadata queries which all go through one of the metadata_all__-files
* Removed unused versionUtil methods, and added a method to convert into the underscored fileVersion format
* Updated API-/service-/repository-layers for metadata
* Updated unit tests accordingly
* Updated configuration. Now instead of hardcoding the entire metadata file-path, the version is concatenated on request.

### CODE REVIEW

File: src/test/java/no/microdata/datastore/adapters/api/MetadataAPITest.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public void testGetDataStructures() throws Exception {
                 "names", List.of("FNR", "AKT_ARBAP"),
                 "languages", "no",
                 "requestId", "56",
-                "version", VersionUtils.toThreeLabelsIfNotDraft(requestVersion),
+                "version", "3_2_1",
```
Comment (pawbu) [inline src/test/java/no/microdata/datastore/adapters/api/MetadataAPITest.java:154]: This breaks the API contract. I think we should use standard way of writing semantic versioning in APIs.

**Review by pawbu — CHANGES_REQUESTED** [review]

File: src/test/java/no/microdata/datastore/adapters/api/MetadataAPITest.java (hunk 1 above)
Comment (DanielElisenberg) [reply src/test/java/no/microdata/datastore/adapters/api/MetadataAPITest.java:154]: It doesn't change anything in the API. (At least not that I can see)
Line 162 in this test uses requestVersion "3.2.1.0", its just transformed to an underscored version in the query (used internally).

Comment (pawbu) [reply src/test/java/no/microdata/datastore/adapters/api/MetadataAPITest.java:154]: You're right. This looked like a query coming from client.

**Review by pawbu — APPROVED** [review]
👍

**Pull request merged**